src/lib/seam/components/EditAccessCodeForm/EditAccessCodeForm.tsx

```diff
@@ -19,6 +19,7 @@ import {
 
 export interface EditAccessCodeFormProps extends CommonProps {
   accessCodeId: string
+  onSuccess?: () => void
```

Comment:
```suggestion
  onSuccess?: (accessCode: AccessCode) => void
```

Reply:
This wouldn't work here since `useUpdateAccessCode` returns an `ActionAttempt<'UPDATE_ACCESS_CODE'>`, not an `AccessCode`.

Reply:
We should keep the same interface for onSuccess. We could either always only pass the access code id, or after creation we can make a get call and return the access code.

Reply:
Maybe we should always do the id as the first argument. That matches props like onAccessCodeClick. We can always add a second argument that receives the full object if that is requested.